### PR TITLE
feat(studio): diagram option to fit the current flow

### PR DESCRIPTION
**Pull request by michaelmass**
## Feature
A new button to fit the current flow into the screen.

Animation

### CODE REVIEW

File: packages/studio-ui/src/web/reducers/ui.ts [hunk 1]
```diff
@@ -92,6 +93,12 @@ const reducer = handleActions(
         zoomLevel: state.zoomLevel + 25
       }
     },
+    [zoomToFit]: (state, {}) => {
+      return {
+        ...state,
+        zoomLevel: -1
```
Comment (laurentlp) [inline packages/studio-ui/src/web/reducers/ui.ts:99]: Can you put this into a reusable constant so we know that this magic number means that the zoom should be adjusted?

**Review by laurentlp — COMMENTED** [review]
Simple enough! Nice work!!!

I have a small comment otherwise LGTM!

**Comment by allardy** [comment]
Please add translations for the buttons in the translation files, eg `en.json`

**Comment by laurentlp** [comment]
> Please add translations for the buttons in the translation files, eg `en.json`

Right, good catch, I forgot to mention it!

**Comment by EFF** [comment]
@laurentlp @allardy  I fixed your comments

**Pull request merged**